drivers/hmis/lib/form_data/tarrant_county/assessments/caseworker_exit.json

```diff
@@ -0,0 +1,563 @@
+{
+  "item": [
+    {
+      "text": "Client Data",
+      "type": "GROUP",
+      "link_id": "caseworker_exit_client_data",
+      "item": [
+        {
+          "text": "Assessment Date",
```

Comment:
We confirmed this should be allowed post exit; should there be some post-exit bound such as 90 days?